src/server_manager/messages/en.json

```diff
@@ -86,18 +86,24 @@
   "feedback-suggestion": "Suggestions",
   "feedback-title-generic": "Send Feedback",
   "feedback-title-install": "Outline Server Installation Failed",
-  "gcp-firewall-assign-0": "Navigate to the {openLink}VM instances screen{closeLink}.",
-  "gcp-firewall-assign-1": "Click on the instance where you want to host Outline.",
```

Comment:
Remove from `master_messages.json`.

Reply:
I think this got messed up in the merge, I didn't want to delete these

Reply:
Done in 870ca84